Makefile.envs

```diff
@@ -1,12 +1,8 @@
 export EMSCRIPTEN_VERSION = 1.38.44
 export BINARYEN_VERSION = version_86
 
-export PATH := $(PYODIDE_ROOT)/ccache:$(PYODIDE_ROOT)/emsdk/emsdk:$(PYODIDE_ROOT)/emsdk/emsdk/node/12.18.1_64bit/bin:$(PYODIDE_ROOT)/emsdk/emsdk/fastcomp/emscripten/:$(PYODIDE_ROOT)/node_modules/.bin/:$(PATH)
-
-export EMSDK = $(PYODIDE_ROOT)/emsdk/emsdk
-export EM_CONFIG = $(PYODIDE_ROOT)/emsdk/emsdk/.emscripten
-export EM_CACHE = $(PYODIDE_ROOT)/emsdk/emsdk/.emscripten_cache
-export BINARYEN_ROOT = $(PYODIDE_ROOT)/emsdk/emsdk/binaryen
+export BASH_ENV := $(PYODIDE_ROOT)/set_env.sh
+SHELL := /bin/bash
```

Comment:
It would be helpful to have a reminder of what BASH_ENV does and indicate the we need to ensure we use bash because of it (and not some other reason).